Apps/Sandcastle/gallery/iModel Mesh Export Service.html

```diff
@@ -31,10 +31,13 @@
       window.startup = async function (Cesium) {
         "use strict";
         //Sandcastle_Begin
-        const serviceResponse = await fetch("https://api.cesium.com/itwin/token");
-        const { access_token: token } = await serviceResponse.json();
+        // Generate a share key for access to an itwin without oauth
+        // https://developer.bentley.com/apis/access-control-v2/operations/create-itwin-share/
```

Comment:
Can we add a link to this in the ref doc as well, like we did for [`GoogleMaps`](https://ci-builds.cesium.com/cesium/itwin-share-key/Build/Documentation/GoogleMaps.html?classFilter=google)?